src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -1,4 +1,3 @@
-
 > *This guide provides essential information to help you select and complete the activities you need for your Amplify project. We organized each section around a distinct job or decision point to help you understand your available options, steps to complete, and recommended best practices. For users who want to learn more, we included extra details and explanations in expandable sections within the guide.*
```

Comment:
General notes
Staged view - I already committed the changes to the main branch so you can see them via the staged view here - https://jtbd-auth.dbsujo8obnykj.amplifyapp.com/lib/auth/set_up_auth/q/platform/js/#add-the-sign-up-sign-in-and-sign-out-capabilities
Expanders - We are still working on the expander so somethings aren't showing correctly yet. So some changes to point out that are WIP - 1) Expanders will be removed from the right menu so the four headings at the top right will be removed; 2) H3 and H4 headings will be aligned so we won't have the difference between Blue and Black headings 3) there is a footer that will be added to the end of the expanders along with slowing that animation on close.

Reply:
understood! thank you for the update